.rubocop_todo.yml

```diff
@@ -1,20 +0,0 @@
-# This configuration was generated by
-# `rubocop --auto-gen-config`
-# on 2020-06-01 19:18:27 -0600 using RuboCop version 0.84.0.
-# The point is for the user to remove these configuration records
-# one by one as the offenses are removed from the code base.
-# Note that changes in the inspected code, or installation of new
-# versions of RuboCop, may require this file to be generated again.
-
-# Offense count: 1
-# Configuration parameters: Include.
-# Include: app/controllers/**/*.rb
-Rails/LexicallyScopedActionFilter:
-  Exclude:
-    - 'app/controllers/user/base_controller.rb'
-
-# Offense count: 1
-# Configuration parameters: MinBodyLength.
-Style/GuardClause:
-  Exclude:
-    - 'app/controllers/user/trips_controller.rb'
```

Comment:
Should be resolved or moved into `.rubocop.yml`

Reply:
ok